src/derivatives/atlas.md

```diff
@@ -341,23 +333,24 @@ A guide for using macros can be found at
             "sub-01_space-MNI305_atlas-AAL_dseg.nii.gz": "",
             "sub-01_space-MNI305_atlas-AAL_dseg.tsv": "",
             "sub-01_space-MNI305_atlas-AAL_probseg.nii.gz": "",
-            "sub-01_space-MNI305_label-brain_mask.nii.gz": "",
-            "sub-01_space-MNI305_label-head_mask.nii.gz": "",
-            "sub-01_space-MNI305_T1w.nii.gz": "",
-            "sub-01_space-MNI305_T1w.json": "",
-            "sub-01_space-T1w_label-brain_mask.nii.gz": "",
-            "sub-01_space-T1w_label-head_mask.nii.gz": "",
-            "sub-01_space-T1w_T1w.nii.gz": "",
-            "sub-01_space-T1w_T1w.json": "",
+            "sub-01_space-MNI305_atlas-AAL_seg-brain_mask.nii.gz": "",
+            "sub-01_space-MNI305_atlas-AAL_seg-head_mask.nii.gz": "",
+            "sub-01_space-MNI305_atlas-AAL_T1w.nii.gz": "",
+            "sub-01_space-MNI305_atlas-AAL_T1w.json": "",
+            "sub-01_space-T1w_atlas-AAL_label-brain_mask.nii.gz": "",
+            "sub-01_space-T1w_atlas-AAL_label-head_mask.nii.gz": "",
+            "sub-01_space-T1w_atlas-AAL_T1w.nii.gz": "",
+            "sub-01_space-T1w_atlas-AAL_T1w.json": "",
```

Comment:
What does this mean? This example wants to show that you can combine results derived from an atlas (AAL, in this case) and results derived from, e.g., applying FSL FAST and extracting the brain and the head mask.

Adding atlas-AAL here is confusing and defeats the purpose of them in the example. Happy to include explanations to make the point clearer.